.github/workflows/client.yml

```diff
@@ -38,6 +38,14 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
+      - name: Load environment variables
+        uses: keep-network/load-env-variables@main # TODO: use @v1 once availible
+        env: 
+          CI_GITHUB_TOKEN: ${{ secrets.CI_GITHUB_TOKEN }}
+        with:
+          environment: 'ropsten'
```

Comment:
Can we leave a `TODO` comment here that once we integrate RFC-18 changes we should pass the environment from workflow_dispatch?

Reply:
I'm not sure if passing environment here is the way to go. There are 2 factors that will need to be taken in mind:
1. When triggering upstream build in `ci` project, we don't provide environment in form of the full name (like `ropsten`), but in form of the id that will be used in versioning of artifacts (like `rc`). And this value is then passed to downstream workflows. So the action and/or config files would need to be modified to support such value.
2. We have workflows which use the `load-env-variables` action for both `ropsten` and `alfajores`. They would need to be split...
I'll add a comment about considering substitution of the `ropsten`/`alfajores` values with `environment` input and we will decide about the approach at later stage.